clients/venice-client/src/main/java/com/linkedin/venice/fastclient/DualReadAvroGenericStoreClient.java

```diff
@@ -128,6 +128,7 @@ private static <T> CompletableFuture<T> dualExecute(
     return valueFuture;
   }
 
+  /* TODO both get(key) and get(ctx,key) fetches non map for vson to start with and needs some type of lazy loading? */
```

Comment:
Can you elaborate on this issue?

Reply:
Slack [thread](https://linkedin-randd.slack.com/archives/G0GEB715Y/p1670440026847499) on this issue.
Filed a bug for the same: [VENG-10197](https://jira01.corp.linkedin.com:8443/browse/VENG-10197).

Reply:
Resolving this, to be taken care in the future PR.